test/unit/payments/get-payments-by-scheme-id.test.js

```diff
@@ -0,0 +1,26 @@
+jest.mock('../../../app/payments/get-data')
+const { getData: mockGetData } = require('../../../app/payments/get-data')
+
+const { SCHEME_ID: SCHEME_ID_CATEGORY } = require('../../../app/constants/categories')
+const { SCHEME_ID_VALUE } = require('../../../app/constants/scheme-id-value')
+
+const { DATA } = require('../../mocks/values/data')
+
+const { getPaymentsBySchemeId } = require('../../../app/payments')
+
+describe('get payments by frn', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
+    mockGetData.mockResolvedValue(DATA)
+  })
+
+  test('should get data with FRN category and value', async () => {
```

Comment:
Payment data? What data are we talking about here?

Reply:
its mock data for the test but have updated the test name to be more clear.